lib/msal-core/src/AadAuthority.ts

```diff
@@ -3,6 +3,7 @@
 
 import { Authority, AuthorityType } from "./Authority";
 import { XhrClient } from "./XHRClient";
+import { AADTrustedHostList } from './Constants';
```

Comment:
I am getting a linting error since you are using single quotes here instead of double quotes, this will cause the build to fail.

Reply:
Fixing this doesn't cause any further build issues.

Reply:
weird ... wonder why it didnt break me!

Reply:
fixing